Modules/Scripted/SegmentEditor/SegmentEditor.py

```diff
@@ -66,6 +66,9 @@ def setup(self):
     self.addObserver(slicer.mrmlScene, slicer.mrmlScene.EndCloseEvent, self.onSceneEndClose)
     self.addObserver(slicer.mrmlScene, slicer.mrmlScene.EndImportEvent, self.onSceneEndImport)
 
+    if hasattr(self.layout, "addStretch"):
+      self.layout.addStretch()
```

Comment:
Why do we need the `hasattr` check? Qt4/Qt5 difference?

Reply:
Because not all qt layout classes have that method and if you use the SegmentEditor in your own module as follows, setup would crash (e.g. QFormLayout and QGridLayout don't have that method):

```
from SegmentEditor import SegmentEditorWidget
widget = qt.QWidget()
widget.setLayout(qt.QGridLayout())
segmentEditor = SegmentEditorWidget(parent=widget)
segmentEditor.setup()
widget.show()
```